content/en/cosign/signing_with_blobs.md

```diff
@@ -95,3 +48,51 @@ tlog entry created with index: 965333
 Signature wrote in the file sig
 Certificate wrote in the file cert.pem
 ```
+
+
+## Signing with a key
+
+You may specify your own keys for signing.  You will need the password for the private key to sign:
+
+```shell
+$ cosign sign-blob --key cosign.key README.md
+Using payload from: README.md
+Enter password for private key:
+MEQCIAU4wPBpl/U5Vtdx/eJFgR0nICiiNCgyWPWarupH0onwAiAv5ycIKgztxHNVG7bzUjqHuvK2gsc4MWxwDgtDh0JINw==
+```
+
+This supports all the same flags and features as `cosign sign`, including KMS support, hardware tokens, and keyless signatures.
+
+## Blobs in OCI Registries
+
+You can upload blobs to an OCI registry (similar to ORAS) where they can then be signed like any other image.
+
+You can publish an artifact with `cosign upload blob`:
+
+```shell
+$ echo "my first artifact" > artifact
+$ cosign upload blob -f artifact gcr.io/user-vmtest2/artifact
```

Comment:
Maybe we can use a standard username here instead of `user-vmtest2`? It can be just `user` for example

Reply:
The other places are using `user/demo`

Reply:
Changed it to user/demo